src/main/kotlin/io/github/anki/anki/controller/model/CardDto.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.anki.anki.controller.model
+import io.github.anki.anki.repository.mongodb.model.MongoCard
+import io.github.anki.anki.service.model.Card
+import org.bson.types.ObjectId
+
+
+class CardDto (
+    var parentCollectionId: String,
+    var cardKey: String,
+    var cardValue: String,
+) {
+
+    private var _id: String? = null
+
+    var id: String?
+        get() = _id
+        set(value) {
+            _id = value
+        }
+
+
+    fun toEntity(): Card {
```

Comment:
Mapper should be external function or class

The entity usually means database model, and this naming related to service layer object little bit confusing.